chain/penumbra/penumbra_app_node.go

```diff
@@ -132,47 +193,69 @@ func (p *PenumbraAppNode) GenerateGenesisFile(
 	if err != nil {
 		return fmt.Errorf("error marshalling validators to json: %w", err)
 	}
+
 	fw := dockerutil.NewFileWriter(p.log, p.DockerClient, p.TestName)
 	if err := fw.WriteFile(ctx, p.VolumeName, "validators.json", validatorsJson); err != nil {
 		return fmt.Errorf("error writing validators to file: %w", err)
 	}
-	allocationsCsv := []byte(`"amount","denom","address"\n`)
+
+	allocationsCsv := []byte("\"amount\",\"denom\",\"address\"\n")
 	for _, allocation := range allocations {
-		allocationsCsv = append(allocationsCsv, []byte(fmt.Sprintf(`"%s","%s","%s"\n`, allocation.Amount.String(), allocation.Denom, allocation.Address))...)
+		allocationsCsv = append(allocationsCsv, []byte(fmt.Sprintf("\"%s\",\"%s\",\"%s\"\n", allocation.Amount.String(), allocation.Denom, allocation.Address))...)
```

Comment:
this is logically the same, but the previous string with the ` is easier to understand

Reply:
this was my thought originally as well but we were seeing an error when attempting to parse the allocation csv file during start up.

```
Error Trace:	/Users/justintieri/go/src/github.com/strangelove-ventures/interchaintest/examples/penumbra/penumbra_chain_test.go:56
Error:      	Received unexpected error:
                 generating genesis file: failed to exec testnet generate: exit code 1: 2023-09-18T18:04:18.424475Z  INFO pd: Generating network config chain_id="penumbra-1"
        	Error: could not parse allocations file "/home/heighliner/allocations.csv"
        	            	
        	Caused by:
        	            CSV error: record 1 (line: 2, byte: 27): found record with 9 fields, but the previous record has 3 fields
```

Reply:
Go's raw strings treat all values literally, so the escape character here throws it off

\`"%s","%s","%s"\`+"\n" - would resolve this

same for allocation if you want: \`"amount","denom","address"\` + "\n"